administrator/components/com_content/src/Model/ArticleModel.php

```diff
@@ -708,7 +708,7 @@ public function save($data)
             $check = $input->post->get('jform', [], 'array');
 
             foreach ($data['urls'] as $i => $url) {
-                if ($url != false && ($i == 'urla' || $i == 'urlb' || $i == 'urlc')) {
+                if ($url && ($i == 'urla' || $i == 'urlb' || $i == 'urlc')) {
```

Comment:
$url is a string, so it would be clearer if we check that it is not empty string
```suggestion
                if (trim($url) !== '' && ($i == 'urla' || $i == 'urlb' || $i == 'urlc'))
```